.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- Thank you for contributing to our WordPress open source project! -->
+
+## Description
+<!-- Please provide a clear and concise description of your changes -->
+<!-- What problem does this PR solve? What functionality does it add/improve? -->
+
+## Related Issue
+<!-- Please link any related issues here. Use the syntax: "Fixes #123" or "Related to #456" -->
+
+## Dependant PRs
+<!-- List any dependent PR's that are awaiting review. Use the following syntax: "#1234" -->
+
+## Type of Change
+<!-- What types of changes does your code introduce? Put an x in all boxes that apply -->
+- [ ] ✅ Bug fix (non-breaking change which fixes an issue)
+- [ ] ✨ New feature (non-breaking change which adds functionality)
+- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] 🧹 Code refactoring (no functional changes)
+- [ ] 📄 Example update (no functional changes)
+- [ ] 📝 Documentation update
+- [ ] 🔍 Performance improvement
+- [ ] 🧪 Test update
```

Comment:
I don't think this type of change section needs to be here. I'm a fan of making these things minimal. Changelogs should include this info and can always be discussed. Open to hearing other opinions

Reply:
I understand your perspective, but let's discuss a bit more "Type of Change" section. 
Having clickable checkboxes provides immediate visibility into the PR's nature without requiring reviewers to parse the entire description. It takes minimal effort for contributors to check a box, yet provides valuable context that helps prioritize reviews and understand potential impacts. The checklist also serves as a helpful reminder for contributors to consider the scope and impact of their changes as a result of their work. While changelogs can include this information later, having it front and center in the PR template improves the initial review process and helps set appropriate expectations for all stakeholders involved.